src/xml2js.coffee

```diff
@@ -169,6 +169,8 @@ class exports.Builder
             if typeof child is 'string' && @options.cdata && requiresCDATA child
               element = element.ele(key).raw(wrapCDATA child).up()
             else
+              if child === undefined || child === null
```

Comment:
You can say `==` here (or `is`), as that compiles to strict equality. Or even better, use `child?`.